os/include/queue.h

```diff
@@ -86,6 +86,13 @@
 #define sq_peek(q)  ((q)->head)
 #define dq_peek(q)  ((q)->head)
 
+#if defined(NXFUSE_HOST_BUILD) && !defined(NULL)
```

Comment:
I don't know why this is necessary. How about linking <sys/types.h> instead of adding duplication?

Reply:
@sunghan-chang: Even we too agree for your point of linking <sys/types.h> for nxfuse host build Mr.Chang to avoid "NULL" definition error.

But <sys/types.h>is a much sensitive file, where the datatypes like int, flot, char etc are defined. which are architecture specific(Say TizenRT is arm specific) and same wont be feasible for Linux-Host x86 arch.

Given above fact, If we try to link that for nxfuse Linux-Host build(x86), lot of redefinition errors are newly cropping, fixing those are not needed as well. Because Linux-Host should use the host PC's <sys/types.h> file for smooth execution.

To avoid the above issues, we declared the "NULL" in queue.h.

ToDo:
Meantime we will also trying to fix this by including proper header for "NXFUSE_HOST_BUILD" where the NULL is defined.